.golangci.yml

```diff
@@ -0,0 +1,60 @@
+# This file configures github.com/golangci/golangci-lint.
+
+run:
+  timeout: 3m
+  tests: true
+  # default is true. Enables skipping of directories:
+  #   vendor$, third_party$, testdata$, examples$, Godeps$, builtin$
+  skip-dirs-use-default: true
+
+service:
+  golangci-lint-version: 1.49.0
+
+linters:
+  disable-all: true
+  enable:
+    - whitespace # Tool for detection of leading and trailing whitespace
+    - wsl # Whitespace Linter - Forces you to use empty lines!
+    - wastedassign # Finds wasted assignment statements.
+    - unconvert # Unnecessary type conversions
+    - tparallel # detects inappropriate usage of t.Parallel() method in your Go test codes
+    - thelper # Detects golang test helpers without t.Helper() call and checks the consistency of test helpers
+    - stylecheck # Stylecheck is a replacement for golint
+    - prealloc # Finds slice declarations that could potentially be pre-allocated
+    - predeclared # Find code that shadows one of Go's predeclared identifiers
```

Comment:
Should be `Finds` instead of `Find`, the other comments are formed in the active verb voice

Reply:
Fixed